.circleci/config.yml

```diff
@@ -1,24 +1,37 @@
 version: 2
 
 jobs:
-
   build:
     docker:
-      - image: python:3.7-alpine
+      - image: circleci/python:3.7
     steps:
       - checkout
-      - run: apk add git
-      - run: pip install -r requirements.txt
-      - run: echo 'Run the tests....'
+
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
+      - restore_cache:
+          key: deps9-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
+
+      - run:
+          command: |
+            sudo pip install pipenv
+            pipenv install --dev
```

Comment:
pipenv is nice, it lets you manage which dependencies should be installed on a dev machine, vs. a production server